src/pages/Animetoast/tests.json

```diff
@@ -0,0 +1,31 @@
+{
+    "title": "Animetoast",
+    "url": "https://www.animetoast.cc/",
+    "testCases": [
+      {
+        "url": "https://www.animetoast.cc/vinland-saga-ger-sub/?link=81",
+        "expected": {
+          "sync": true,
+          "title": "Vinland Saga",
+          "identifier": "vinland-saga-ger-sub",
+          "overviewUrl": "https://www.animetoast.cc/vinland-saga-ger-sub",
+          "episode": 10,
+          "nextEpUrl": "https://www.animetoast.cc/vinland-saga-ger-sub/?link=82",
+          "uiSelector": true
+        }
+      },
+      {
+        "url": "https://www.animetoast.cc/one-piece/?link=163",
```

Comment:
Two times a test for the video page and not for the overview. 
Example:
https://github.com/MALSync/MALSync/blob/master/src/pages/Gogoanime/tests.json